### DISCUSSION TITLE
how to implement an empty submenu

### DISCUSSION THREAD

**Discussion by hgfrey** [primefaces/community#2153] [PrimeFaces] [answered]
Is there any suggestion on how to implement an empty submenu?

**Comment by melloware** [comment]
What is your use case?

**Reply by hgfrey** [reply, marked as answer]
We have a main menu and a submenu. There are not always menu items in the submenu (see the attached images). We are currently displaying an empty submenu by adding an item that has no text. This is fine as far as it goes, but it does lead to minor discrepancies. With PF 14 the empty element is suddenly displayed black and the screen reader reads the element as empty. It's not all that bad, but I'm wondering if there's a more elegant way. 
Thank you

![image](https://github.com/primefaces/community/assets/33621098/2d17ae66-7aed-4f7b-b68f-75574ec9a850)

![image](https://github.com/primefaces/community/assets/33621098/873638f5-dc40-4f14-9261-7c1c6006f2f0)

![image](https://github.com/primefaces/community/assets/33621098/eed25d69-659b-4539-8e06-dc9a9ec6f6e1)

**Reply by melloware** [reply]
No your way seems to be the only way.

**Reply by hgfrey** [reply]
Understood. Then I adjust our CSS styles.
